src/providers/cortex/trulens/providers/cortex/provider.py

```diff
@@ -17,13 +19,15 @@ class Cortex(
     DEFAULT_MODEL_ENGINE: ClassVar[str] = "snowflake-arctic"
 
     model_engine: str
+    endpoint: CortexEndpoint = pydantic.Field(exclude=True)
+    snowflake_conn: SnowflakeConnection
```

Comment:
endpoint and snowflake_conn are both declared again below under the docstring as well

Reply:
fixed now - part of merge mess ahah